relbench/data/task_node.py

```diff
@@ -93,3 +107,34 @@ def __init__(self, dataset: str, process: bool = False) -> None:
 
         def pack_tables(self, root: Union[str, os.PathLike]) -> Tuple[str, str]:
             return _pack_tables(self, root)
+
+    def stats(
```

Comment:
Number of trues and number of false for binary
regression: min, max and mean etc.